libcontainer/configs/memorypolicy.go

```diff
@@ -0,0 +1,29 @@
+package configs
+
+import "golang.org/x/sys/unix"
+
+//nolint:revive,staticcheck,nolintlint // ignore ALL_CAPS errors in consts from numaif.h, will match unix.* in the future
+const (
+	MPOL_DEFAULT = iota
+	MPOL_PREFERRED
+	MPOL_BIND
+	MPOL_INTERLEAVE
+	MPOL_LOCAL
+	MPOL_PREFERRED_MANY
+	MPOL_WEIGHTED_INTERLEAVE
```

Comment:
nit: using iota here is no good. The values of these constants are well know and they should be explicitly provided here.